metadata/user_feedsMetaData.php

```diff
@@ -41,17 +41,41 @@
  * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  */
 
-$dictionary['users_feeds'] = array( 'table' => 'users_feeds'
-                                  , 'fields' => array(
-    
-       array('name' =>'user_id', 'type' =>'varchar', 'len'=>'36', )
-      , array('name' =>'feed_id', 'type' =>'varchar', 'len'=>'36', )
-      , array('name' =>'rank', 'type' =>'int', 'required' => false)
-      , array('name' => 'date_modified','type' => 'datetime')
-      , array('name' =>'deleted', 'type' =>'bool', 'len'=>'', 'default'=>'0', 'required' => false)
-                                                      )
-                                 , 'indices' => array(
-  
-       array('name' =>'idx_ud_user_id', 'type' =>'index', 'fields'=>array('user_id', 'feed_id'))
-                                                      )
-                                  );
+$dictionary['users_feeds'] = [
+    'table' => 'users_feeds',
+    'fields' => [
+        [
+            'name' =>'user_id',
+            'type' =>'varchar',
+            'len'=>'36',
+        ],
+        [
+            'name' =>'feed_id',
+            'type' =>'varchar',
+            'len'=>'36',
+        ],
+        [
+            'name' => 'suite_rank',
```

Comment:
'ranking' would maybe be a better option over 'suite_rank', overall this is a breaking change